packages/core/src/checkbox/Checkbox.css

```diff
@@ -4,6 +4,7 @@
   gap: var(--salt-spacing-100);
   position: relative;
   cursor: var(--salt-selectable-cursor-hover);
+  align-items: center;
```

Comment:
Check with design, i think the fix is to put spacing on top to accommodate multi-line label use cases. i.e. the box should be alined with first line of text only

Reply:
maybe also add multi-line use cases to Chromatic snapshot